vaadin-upload-flow-parent/vaadin-upload-flow/src/main/java/com/vaadin/flow/component/upload/UploadI18N.java

```diff
@@ -421,6 +421,8 @@ public static class Units implements Serializable {
         private List<String> size = Arrays.asList("B", "kB", "MB", "GB", "TB",
                 "PB", "EB", "ZB", "YB");
 
+        private Integer sizeBase;
```

Comment:
I wonder why `size` needs a default value. I notice that we tend to avoid default values in i18n to keep those from the web component. But since this one has, I wonder if it would make sense for sizeBase to have it too.

Reply:
My reasoning to keep `sizeBase` without an explicit default value in the Flow counterpart was exactly that. The default value is already handled in the web component and an i18n configuration without an explicit `sizeBase` value is supported. As per why there is one for `size`, I cannot think of any good reason. The value is exactly the same as in the web component. The difference might be that it is a required parameter and the fall back is not handled as it is done with the `sizeBase`.

Reply:
As discussed internally, the size property has a default value apparently due to an oversight during review when it was contributed. The correct approach is to let the web component handle the default values for i18n propertiesl